docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -3661,6 +3661,10 @@ The `limits` block configures default and per-tenant limits imposed by component
 # CLI flag: -alertmanager.max-config-size-bytes
```

Comment:
Just two lines above this one, the old description for the config size limit should be fixed.

Reply:
This is the description of the remote AM configuration limit. I had copied the grafana config limit from this, that's why it's almost the same 😅 
I've adjust it now to keep the same wording for both